python/ct/crypto/verify_ecdsa.py

```diff
@@ -26,22 +24,27 @@ def __init__(self, key_info):
 
         Raises:
         - PemError: If the key has an invalid encoding
+        - UnsupportedAlgorithmError: If the key uses an unsupported algorithm
         """
         if (key_info.type != client_pb2.KeyInfo.ECDSA):
             raise error.UnsupportedAlgorithmError(
                 "Expected ECDSA key, but got key type %d" % key_info.type)
 
-        # Will raise a PemError on invalid encoding
-        self.__der, _ = pem.from_pem(key_info.pem_key, self.__READ_MARKERS)
```

Comment:
No, because the cryptography library can load a public key directly from PEM, so there's no need to manually convert to DER first. It can also convert straight back to PEM, so self.__der isn't required in __repr__() either.